yarn/src/test/scala/org/apache/spark/scheduler/cluster/StubApplicationAttemptId.scala

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Can we put these fake stub class into one file like: `SparkYarnTestHelper` or something else? That will possibly reduce the file number.